addon/src/lib/reducers/sideEffects.js

```diff
@@ -181,6 +183,20 @@ export function reducer(
         installManager.syncInstalled();
       };
 
+    case actions.BROWSER_STARTUP.type:
```

Comment:
I don't know what's better, having a startup action like this, or just putting unconditional initialization in `main.js`? So far I've gone with main. You'll see I've already got the GA ping there.